PubNub/Core/PubNub+Publish.m

```diff
@@ -88,6 +89,10 @@ - (void)  publish:(id)message toChannel:(NSString *)channel
     [self publish:message toChannel:channel compressed:NO withCompletion:block];
```

Comment:
@jzucker2 because we have new designated method, we can rewrite this method like this:

_[self publish:message toChannel:channel withMetadata:nil withCompletion:block];_